Doc/whatsnew/3.11.rst

```diff
@@ -233,6 +233,13 @@ sqlite3
   :issue:`16379`.)
 
 
+unicodedata
+-----------
+* If the given string is pure ASCII string, :func:`unicode.normalize` now
+  handles this as an already normalized to process it in constant time.
+  (Contributed by Dong-hee Na in :issue:`bpo-44987`.)
```

Comment:
It is not worth a new subsection in the "Improved Modules" section. An entry in the "Optimizations" section should be enough. And make it shorter. E.g. "Pure ASCII strings are now normalized in constant time."